### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by BelehovEgor**
- edit user (removed roleId, startWorkAt now can edit only user with rights)
- removed exceptions from credentials repository

### CODE REVIEW

File: src/UserService.Mappers/Models/PatchDbUserMapper.cs [hunk 1]
```diff
@@ -5,63 +5,57 @@
...
+        }
+        if (item.path.EndsWith(nameof(EditUserRequest.DateOfBirth), StringComparison.OrdinalIgnoreCase)
+          || item.path.EndsWith(nameof(EditUserRequest.StartWorkingAt), StringComparison.OrdinalIgnoreCase))
+        {
+          result.Operations.Add(new Operation<DbUser>(item.op, item.path, item.from, DateTime.TryParse(value(item), out DateTime date) ? date : null));
+          continue;
+        }
+        if (item.path.EndsWith(nameof(EditUserRequest.City), StringComparison.OrdinalIgnoreCase))
+        {
+          result.Operations.Add(new Operation<DbUser>(item.op, item.path, item.from, value(item)));
```
Comment (MariyaSin) [inline src/UserService.Mappers/Models/PatchDbUserMapper.cs:51]: same as 55 line

**Review by MariyaSin — APPROVED** [review]

File: src/UserService.Business/Commands/User/EditUserCommand.cs [hunk 2]
```diff
@@ -71,37 +69,6 @@ private async Task EditUserOfficeAsync(Guid? officeId, Guid userId, List<string>
       errors.Add(officeErrorMessage);
     }
 
-    private async Task ChangeUserRoleAsync(Guid? roleId, Guid userId, List<string> errors)
```
Comment (MariyaSin) [inline src/UserService.Business/Commands/User/EditUserCommand.cs:74]: do you remove endpoint from appsetings.json and madels.dll?

Comment (BelehovEgor) [reply src/UserService.Business/Commands/User/EditUserCommand.cs:74]: this request is still using in create user command

File: src/UserService.Mappers/Models/PatchDbUserMapper.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/UserService.Mappers/Models/PatchDbUserMapper.cs]: removed this if

**Review by VLiamin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

**Pull request merged**